ssn/rdf/examples/paleo-atmosphere.ttl

```diff
@@ -16,20 +16,20 @@ ex:Ob873t2 a sosa:Observation ;
   sosa:observedProperty ex:C14-Age ;
   sosa:hasFeatureOfInterest ex:Bubble873 ;
   sosa:hasUltimateFeatureOfInterest ex:EarthAtmosphere ;
-  sosa:hasResult "7530"^^unit:YR ;
+  sosa:hasSimpleResult "7530"^^unit:YR ;
```

Comment:
In my understanding this is not legal RDF, although I may not necessarily be a reliable parser.

Please also check discussion in issue #298.

Reply:
See response https://github.com/w3c/sdw-sosa-ssn/issues/298#issuecomment-2896895801